buildconfig-for-dtkimage.yaml

```diff
@@ -0,0 +1,40 @@
+#apiVersion: project.openshift.io/v1
+#kind: Project
+#metadata:
+#  annotations:
+#    openshift.io/description: ""
+#    openshift.io/display-name: ""
+#  labels:
+#    kubernetes.io/metadata.name: dtk
+#  name: dtk
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    app: dtk
+  name: dtk
+  namespace: dtk
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
```

Comment:
In which cases we will need to build DTK in-cluster? AFAIK, DTK is shipped with OCP, therefore, each OCP version should already have its DTK image published.